### PR TITLE
Update to V1 APIs and Add RLS Support

### PR DESCRIPTION
**Pull request by aluong**
-Updated CLI to use new V1 APIs which use AppKey auth
-Added support RLS support for embed token creation

### CODE REVIEW

File: lib/cli-create-embed-token.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return val.split(',');
+    }
+
+    program.version(pkg.version)
+        .option('-c, --collection <collection>', 'The Power BI workspace collection')
+        .option('-k, --accessKey <accessKey>', 'The Power BI workspace collection access key')
+        .option('-w, --workspace <workspaceId>', 'The Power BI workspace id')
+        .option('-r, --report <reportId>', 'The Power BI report id')
+        .option('-u, --username [username]', 'The RLS username')
+        .option('--roles [roles]', 'The RLS roles', list);
```
Comment (wbreza) [inline lib/cli-create-embed-token.ts]: Nice :)  This will auto take the param arg and split it and set the value?

**Comment by wbreza** [comment]
LGTM

File: lib/cli-create-embed-token.ts (hunk 1 above)
Comment (mattmazzola) [inline lib/cli-create-embed-token.ts]: I think you should explicitly state in the help and example that this accepts a comma separated list of roles.

Comment (aluong) [inline lib/cli-create-embed-token.ts]: I added it in the example

**Pull request merged**